doc/articles/features/accelerometer.md

```diff
@@ -0,0 +1,67 @@
+# Accelerometer
+
+> [!TIP]
+> This article covers Uno-specific information for Accelerometer. For a full description of the feature and instructions on using it, consult the UWP documentation: https://docs.microsoft.com/en-us/windows/uwp/devices-sensors/use-the-accelerometer
+
+ * The `Windows.Devices.Sensors.Accelerometer` class allows measuring linear acceleration of the device along three X, Y and Z axis. 
+
+## Supported features
+
+| Feature        |  Windows  | Android |  iOS  |  Web (WASM)  | macOS | Linux (Skia)  | Win 7 (Skia) | 
+|---------------|-------|-------|-------|-------|-------|-------|-|
+| `GetDefault`         | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+| `ReadingChanged` | ✔ | ✔ | ✔ | ✔ | ✖ | ✖| ✖ |
+| `Shaken`     | ✔ | ✔ | ✔ | ✔ | ✖ | ✖ | ✖ |
+| `ReportInterval`     | ✔ | ✔ | ✔ | ✔ | ✖ | ✖ | ✖ |
+
+## Using Accelerometer with Uno
+ 
+ * The `GetDefault` method is availble on all targets and will return `null` on those which do not support `Accelerometer` or devices which do not have such sensor.
+* Ensure to unsubscribe from the `ReadingChanged` and `Shaken` events when you no longer need the readings, so that the sensor is no longer active to avoid unnecessary battery consumption.
+ * While iOS features a built-in shake gesture recognition, this is not available on Android and WASM. For these platforms we use a commonly used implementation which tries to approximate the shake motion to detect it. Both Android and WASM use the same implementation. In case this is not sufficient for your use case, you can implement Shake detection utilizing the `ReadingChanged` event readings.
+ * On Android, when both `ReadingChanged` and `Shaken` events are attached and the user sets the `ReportInterval` to a high value, the `ReadingChanged` event may be raised more often than requested. This is because for multiple subscribers to the same sensor the system may raise the sensor events with the frequency of the one with lower requested report delay. This is, however, in line with the behavior of UWP Accelerometer and you can filter the events as necessary for your use case.
```

Comment:
```suggestion
 * On Android, when both `ReadingChanged` and `Shaken` events are attached and the user sets the `ReportInterval` to a high value, the `ReadingChanged` event may be raised more often than requested. This is because for multiple subscribers to the same sensor the system may raise the sensor events with the frequency of the one with a lower requested report delay. This is, however, in line with the behavior of the UWP Accelerometer and you can filter the events as necessary for your use case.
```